db/migrate/20210226072406_add_not_null_constraint_to_packages.rb

```diff
@@ -0,0 +1,5 @@
+class AddNotNullConstraintToPackages < ActiveRecord::Migration[5.2]
+  def change
+    change_column_null :packages, :value_hk_dollar, false
```

Comment:
I hadn't thought about the box/pallet case as they have no valuation. What should we do about that?

Reply:
For box and pallet, valuation calculation sets value as 0.0 hence it will be handled.